init-tests-after-clone.sh

```diff
@@ -1,7 +1,9 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
+
+set -e
 
 if [[ -z "$TRAVIS" ]]; then
-  read -p "This operation will destroy locally modified files. Continue ? [N/y]: " answer
+  read -rp "This operation will destroy locally modified files. Continue ? [N/y]: " answer
```

Comment:
I've noticed I never said in the commit message or anywhere else why I added `-r` here. It's recommended to pass `-r` to `read`, except in the rare case that `read`'s very specific backslash interpretation behavior without `-r` is desired, which it's not here. My immediate motivation for adding `-r` was to get a clean [`shellcheck`](https://www.shellcheck.net/) run as a safeguard against accidentally breaking the script while making the changes at the top. Since I believe passing `-r` is an improvement, I left it in.